docs/AboutUs.md

```diff
@@ -15,40 +14,49 @@ You can reach us at the email `e0406389@u.nus.edu`
 
 [[github](https://github.com/JayChenYJ)]
 [[portfolio](team/jaychenyj.md)]
+You can reach me at the email `e0406389@u.nus.edu`
 
-* Role & Responsibilities: Team Lead, Scheduling and tracking, Deliverables and deadlines
-
+* Role: Team Lead
+* Responsibilities: Scheduling and tracking, Deliverables and deadlines
+  
 ### Chew Wei Hao
 
 <img src="images/chewwh09.png" width="200px">
 
 [[github](http://github.com/chewwh09)]
 [[portfolio](team/chewwh09.md)]
+You can reach me at the email `e0417962@u.nus.edu`
 
-* Roles and Responsibilities: Logic expert, Code Quality, Documentation
+* Role: Developer
+* Responsibilities: Logic expert, Code Quality, Documentation
 
 ### Mohamed Fazil
 
 <img src="images/md-fazil.png" width="200px">
 
-[[github](http://github.com/Md-Fazil)] [[portfolio](team/mohamedfazil.md)]
+[[github](http://github.com/Md-Fazil)] [[portfolio](team/mohamedfazil.md)] You can reach me at the email `mohamedfazil@u.nus.edu`
```

Comment:
Perhaps, the line "You can reach me at the email `mohamedfazil@u.nus.edu`" should be in the following line.

Reply:
they will all appear in the same line so I don't think it matters